config/v1/types_cluster_version.go

```diff
@@ -282,11 +288,13 @@ var ClusterVersionCapabilitySets = map[ClusterVersionCapabilitySet][]ClusterVers
 		ClusterVersionCapabilityOpenShiftSamples,
 		ClusterVersionCapabilityBaremetal,
 		ClusterVersionCapabilityMarketplace,
+		ClusterVersionCapabilityInsights,
```

Comment:
this shouldn't go into the 4.11 set, you're doing this for 4.12, right?  So copy the 4.11 set to create a 4.12 set, and add your entry there.

Reply:
@bparees done. I guess... I wasn't 100% sure if I had to replicate the ones from `4.11` but I did anyway... so, if any change is necessary just let me know.